src/CourseAuthoringRoutes.jsx

```diff
@@ -6,7 +6,7 @@ import { PageRoute } from '@edx/frontend-platform/react';
 import CourseAuthoringPage from './CourseAuthoringPage';
 import { PagesAndResources } from './pages-and-resources';
 import ProctoredExamSettings from './proctored-exam-settings/ProctoredExamSettings';
-import DiscussionAppList from './pages-and-resources/discussions/DiscussionAppList';
+import DiscussionsRoutes from './pages-and-resources/discussions/DiscussionsRoutes';
```

Comment:
We have more than one route in the discussions module now, so it gets its own Routes file.